deploy/sre-prometheus/100-cluster-version-operator.PrometheusRule.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-cluster-version-operator
+    role: alert-rules
+  name: sre-cluster-version-operator
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-cluster-version-operator
+    rules:
+    - alert: ClusterOperatorDegraded
+      annotations:
+        summary: Cluster operator has been degraded for 1 day.
+        description: The {{ "{{ $labels.name }}" }} operator is degraded because {{ "{{ $labels.reason }}" }}, and the components it manages may have reduced quality of service.  Cluster upgrades may not complete. For more information refer to 'oc get -o yaml clusteroperator {{ "{{ $labels.name }}" }}'{{ "{{ with $console_url := \"console_url\" | query }}{{ if ne (len (label \"url\" (first $console_url ) ) ) 0}} or {{ label \"url\" (first $console_url ) }}/settings/cluster/{{ end }}{{ end }}" }}.
+      expr: |
+        max by (namespace, name, reason)
+        (
+          (
+            cluster_operator_conditions{job="cluster-version-operator", condition="Degraded", name =~ "kube-apiserver|kube-controller-manager|kube-scheduler"}
+            or on (namespace, name)
+            group by (namespace, name) (cluster_operator_up{job="cluster-version-operator", name =~ "kube-apiserver|kube-controller-manager|kube-scheduler"})
+          ) == 1
+        )
+      for: 1d
+      labels:
+        severity: warning
+        needs_attention: "true"
```

Comment:
We should definitely document this label in ops-sop, in case this is not something you're tracking in your epic yet.

Reply:
https://github.com/openshift/ops-sop/pull/2947